webpack.config.js

```diff
@@ -66,6 +68,7 @@ const adminJsConfig = {
     TimelineEditorScript: './assets/src/blocks/Timeline/TimelineEditorScript.js',
     SocialMediaEditorScript: './assets/src/blocks/SocialMedia/SocialMediaEditorScript.js',
     GalleryEditorScript: './assets/src/blocks/Gallery/GalleryEditorScript.js',
+    HubspotFormEditorScript: './assets/src/blocks/HubspotForm/HubspotFormEditorScript.js',
```

Comment:
Don't we also need to add the CSS files here in the config?

Reply:
What's the main reason to add it also in the config?

Reply:
We do it for other blocks in the [cssConfig](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/79bc8812ac1b9cfa4e12ef77c5b288c0d489efb5/webpack.config.js#L74), I thought it was needed to properly split up the blocks' code

Reply:
Gotcha! I'll move it.

Just to know, what's the difference between adding to Webpack.config instead of [blocks.scss](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/641/files#diff-cbca468115dd29ef21fbf296256aaf045b3c5fa6d03d09caa7a41ff222f315d4R12)?

Reply:
I might be wrong, but I think that when adding them there we only load the styles when the block is actually present on the page, instead of always loading them - so it's better for performance. But don't quote me on that 😬 @Inwerpsel probably knows better!

Reply:
Entirely correct @mleray